src/app/series/directives/series-feeds.component.html

```diff
@@ -230,6 +246,22 @@ <h2>No feeds found! Please contact support</h2>
             </div>
           </prx-fancy-field>
 
+          <prx-fancy-field label="Engagement Settings">
```

Comment:
Note: there's a bunch of unrelated linting going on in this file.  (I didn't have Prettier setup correctly the last time I edited it).

But this is the new section of code.  Note that it shows up on _all_ feeds, even the default.

Question: is the correct place in the form for these checkboxes?